src/app/code/community/Shopgate/Cloudapi/etc/config.xml

```diff
@@ -38,6 +38,14 @@
             </shopgate_cloudapi_setup>
         </resources>
         <events>
+            <controller_action_layout_load_before>
```

Comment:
I am assuming this can be placed into the frontend scope as it only pertains to it.